flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/api/windows.scala

```diff
@@ -20,90 +20,180 @@ package org.apache.flink.table.api
 
 import org.apache.flink.table.expressions._
 import org.apache.flink.table.plan.logical._
-import org.apache.flink.table.typeutils.{RowIntervalTypeInfo, TimeIntervalTypeInfo}
-import org.apache.flink.table.api.scala.{CURRENT_RANGE, CURRENT_ROW}
 
 /**
-  * Over window is similar to the traditional OVER SQL.
+  * A over-window specification.
```

Comment:
If we using `over-window` we should keep consistent throughout the documentation comments. And i think `over-window` is better than `over window` in the Over-window definition.

Reply:
Thanks for the hint. Yes, we should be consistent. Actually, this was the only remaining of `over-window`. I changed it to `over window`. Now we should be consistent with `tumbling windows, session windows` etc.